src/CMakeLists.txt

```diff
@@ -144,6 +145,7 @@ set(qpid_dispatch_LIBRARIES
   ${Proton_Tls_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT}
   ${rt_lib}
+  -lbfd
```

Comment:
Is this necessary for builds that do not have QD_MEMORY_DEBUG enabled?  It would be best to avoid unnecessary dependencies for non-debug builds.